whiteboard/whiteboard-ps/whiteboard/Get-WhiteboardsForTenant.md

```diff
@@ -0,0 +1,149 @@
+---
+external help file: WhiteboardAdmin-help.xml
+Module Name: WhiteboardAdmin
+online version: https://learn.microsoft.com/powershell/module/whiteboard/get-whiteboardsfortenant
+applicable: Microsoft Whiteboard
+title: Get-WhiteboardsForTenant
+schema: 2.0.0
+author: srajabi
+ms.author: shlevari
+ms.reviewer:
+---
+
+# Get-WhiteboardsForTenant
+
+## SYNOPSIS
+
+Gets all the whiteboards associated with a tenant in a specified geography.
+
+## SYNTAX
+
+```powershell
+Get-WhiteboardsForTenant [-Geography] <String>
+  [-IncrementalRunName] <String>
+  [-ForceAuthPrompt]
+  [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+Gets all the whiteboards in a tenant in a specified geography. Returns a list of whiteboard objects. The data is pre-calculated approximately every two weeks and is not realtime.
+
+## EXAMPLES
+
+### EXAMPLE 1
+
+```powershell
+PS C:\> Get-WhiteboardsForTenant -Geography Europe
+```
+
+Gets all the whiteboards associated with the caller's tenant in Europe as a list of whiteboard metadata objects.
+
+### Output
```

Comment:
Please check how to format the yaml code to include the output in the following link:
Output

Reply:
Thanks for reviewing @dariomws. Applied changes to example output based on the document. The cmdlet is already included, just haven't been documented.